src/components/Table/Table.less

```diff
@@ -49,13 +49,55 @@
 	}
 
 	// Table header styles
+	&.lucid-Table-has-light-header > &-Thead {
```

Comment:
it seems like there's a lot of repetition of styles not strictly related to the light-header. Possible to collapse this with the non-light-header table styles?